gulpfile.js

```diff
@@ -13,177 +13,178 @@ USWDS SASS GULPFILE
 
 const autoprefixer = require("autoprefixer");
 const csso = require("postcss-csso");
-const gulp = require("gulp");
-const pkg = require("./node_modules/uswds/package.json");
+const { src, dest, series, parallel, watch } = require("gulp");
+const pkg = require("../uswds/package.json").version;
 const postcss = require("gulp-postcss");
 const replace = require("gulp-replace");
 const sass = require("gulp-sass")(require("sass"));
 const sourcemaps = require("gulp-sourcemaps");
-const uswds = require("./node_modules/uswds-gulp/config/uswds");
 const del = require("del");
 const svgSprite = require("gulp-svg-sprite");
 const rename = require("gulp-rename");
+const log = console.log;
+const colorBlue = "\x1b[34m%s\x1b[0m";
+
 
 /*
 ----------------------------------------
 PATHS
 ----------------------------------------
-- All paths are relative to the
-  project root
-- Don't use a trailing `/` for path
-  names
-----------------------------------------
 */
+let paths = {
+  src: {
+    uswds: `node_modules/uswds/dist`,
+    sass: `node_modules/uswds/dist/scss/theme/**/**`,
+    fonts: `node_modules/uswds/dist/fonts/**/**`,
+    img: `node_modules/uswds/dist/img/**/**`,
+    js: `node_modules/uswds/dist/js/**/**`,
+  },
+  /**
+   * ? project paths
+   * - all paths are relative to the project root
+   * - don't use a trailing `/` for path
+   */
+  dist: {
```

Comment:
This gets exported in `line:175` because we should expect USWDS to be installed via NPM.

Users can easily override by adding their own `dist` object or overriding specific ones:
```js
const uswdsTasks = require("uswds-gulp");

// Where I want to place compiled SCSS
uswdsTasks.paths.css = './my-project-dist/css/'
uswdsTasks.paths.img = './images/`
```